markdownlint.js

```diff
@@ -162,12 +163,17 @@ function printResult(lintResult) {
   }
 
   if (program.output) {
-    try {
-      fs.writeFileSync(program.output, lintResultString);
-    } catch (error) {
-      console.warn('Cannot write to output file ' + program.output + ': ' + error.message);
-      process.exitCode = 2;
-    }
+      try {
```

Comment:
There looks to be an extra indent for this block of lines - combined with my other comment here, I think there's just a single line change?